angr/exploration_techniques/loop_seer.py

```diff
@@ -0,0 +1,140 @@
+import angr
```

Comment:
I would avoid doing `import angr` in angr's code. Please use relative import instead.